zone/client_packet.cpp

```diff
@@ -15205,3 +15315,37 @@ void Client::Handle_OP_ResetAA(const EQApplicationPacket *app)
 	}
 	return;
 }
+/// <summary>
```

Comment:
Please get rid of these unnecessary tags

Reply:
sorry, in VS those tags define on a tooltip when you are typing a call, was added in response to drama, removed all unneeded commenting.